onnxruntime/python/providers/tvm/ort.py

```diff
@@ -5,45 +5,27 @@
 # --------------------------------------------------------------------------
 
 import os
-import timeit
-import numpy as np
 import collections
+import copy
+import logging
 
 import onnx
 import tvm
 from tvm import relay, auto_scheduler
+from tvm.relay import vm
 from tvm.contrib import graph_executor
 from tvm import autotvm
 
+log = logging.getLogger("TVM EP python API logger")
```

Comment:
A name with no space is better is the logs are parsed.

Reply:
Thanks! done